gameplay/src/Quaternion.cpp

```diff
@@ -219,6 +236,11 @@ void Quaternion::set(const Quaternion& q)
     this->w = q.w;
 }
 
+void Quaternion::set(const Vector3& euler)
```

Comment:
Don't add this set method. We don't add a specialized setter for this operation.